### ISSUE TITLE
Question about VT project

### ISSUE THREAD

**Issue by jmalak** [Bill-Gray/PDCurses#154]
I tested build VT project for DOS by Open Watcom and it is build succesfuly for 16/32-bit DOS.

I have a question for which target platform must be VT project build, because it is limited in CMake to UNIX like target only?

**Comment by GitMensch** [comment]
As far as I know it works fine for Win32 and GNU/Linux (both compiled with GCC) and I think it should also work for DOS (never tested this one). The CI builds of VT should definitely be extended, PRs welcome.

**Comment by jmalak** [comment]
if I understand correctly your build system then it should be build on appveyor as standalone build for DOS and as part of Windows build. It is important to know it to be able modify appropriate CMake definition properly.

**Comment by GitMensch** [comment]
As the appveyor integration is currently broken (something to look at for @Bill-Gray) I've mirrored it.
Here's the current built, which may (not) help:
https://ci.appveyor.com/project/GitMensch/pdcurses/builds/33383048

... but as you already have https://ci.appveyor.com/project/jmalak/pdcurses this seems to be not necessary for you.

The VT port _should_ be built:

* with MSC environments (along to wincon and wingui)
* with ow win32 targets (along to wincon and wingui)
* with ow for dos16 and dos32 (along to dos and dosvga; works according to https://github.com/Bill-Gray/PDCurses/blob/master/vt/README.md)
* possibly with ow for linux target (this cross-compilation is currently missing)

**Comment by jmalak** [comment]
Now build is fragmented to DOS, DOSVGA, VT, WINCON+WINGUI, OS2, SDL projects
I suppose it should be build together per target.
OS2 build OS2 only ?
DOS build should be DOS+DOSVGA+VT?
Windows build should be WINCON+WINGUI+VT.
Linux VT+???
What about SDL?

**Comment by GitMensch** [comment]
> I suppose it should be build together per target.

Yes, exactly as you wrote. I _guess_ VT does not work on OS2, @Bill-Gray does it?
Linux VT is enough for now (the otehr options x11 and sdl need too much headers).

I'd leave SDL builds "as is" for now.

**Comment by jmalak** [comment]
Problem with VT is that it refer to some kind of TCP/IP stuff that it must be compile and link against some TCP/IP stack. It compiled for me because I forgot removed Windows headers, my mistake.

**Comment by Bill-Gray** [comment]
I've built and run the VT platform on Linux,  FreeBSD,  Win10,  and DOS.  I haven't tried it on OS/2.  (I've tried the Embarcadero flavor of OS/2,  but it didn't seem to like the hardware.  I didn't try very hard to puzzle it out.)

**Comment by GitMensch** [comment]
Is there anything open or can it be closed? What was with the TCP/IP stuff?
